### DISCUSSION TITLE
How to set up notes

### DISCUSSION THREAD

**Discussion by encomun** [emacs-citar/citar#722] [Q&A] [answered]
Hey,

This is a very beginner question, so my apologies. But I'm having trouble setting up the notes feature. When I go to create a note for a key, I get the message "Make sure ‘citar-notes-paths’ and ‘citar-file-note-extensions’ are non-nil." I did my best to look up how to set those and this is what I came up with:

```
(setq citar-notes-path '("/home/dissertation/Nextcloud/Library/Notes"))
(setq citar-file-note-extensions '("org" "md"))
```

Still getting the same message. Can someone tell me the best way to set this up in my init file? Thanks!

**Comment by bdarcus** [comment, marked as answer]
Assuming that's copy-and-pasted from your init, there's a typo in that first variable (should be plural).

**Reply by encomun** [reply]
Yep, my mistake. Thanks!
